From 60cf6f03a4b13ec0e491a282ab5233a1619a7a66 Mon Sep 17 00:00:00 2001
From: Michael Tremer <michael.tremer@ipfire.org>
Date: Tue, 24 Apr 2018 12:30:13 +0100
Subject: [PATCH] net.c: Include string.h

Various functions that have been used come from string.h. GCC compiled
dma without this header, but unfortunately the binary segfaulted at random
times.

Signed-off-by: Michael Tremer <michael.tremer@ipfire.org>
---
 net.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net.c b/net.c
index a1cc3e3bfd79..221dda131a23 100644
--- a/net.c
+++ b/net.c
@@ -53,6 +53,7 @@
 #include <netdb.h>
 #include <setjmp.h>
 #include <signal.h>
+#include <string.h>
 #include <syslog.h>
 #include <unistd.h>
 
-- 
2.14.3

